src/OpenDB/test/gcd_abstract_lef.lefok

```diff
@@ -0,0 +1,835 @@
+VERSION 5.8 ;
+BUSBITCHARS "[]" ;
+DIVIDERCHAR "/" ;
+UNITS
+    DATABASE MICRONS 1000 ;
+END UNITS
```

Comment:
`_dist_factor` is just being set using its default value, which isn't necessarily wrong, but it's inconsistent with the input DEF (2000).

I think `_dist_factor` should probably just be removed in favor of `tech->getDbUnitsPerMicron()`

Reply:
So now I'm now setting _dist_factor = 1.0L / tech->getDbUnitsPerMicron(), but that appears to not be respected when the DATABASE MICRONS = 2000.

Is there something I don't understand here.

UNITS 1000
![image](https://user-images.githubusercontent.com/3784748/125837983-ccbb6128-45e3-4684-9e8a-f131b80fe92a.png)

UNITS 2000
![image](https://user-images.githubusercontent.com/3784748/125837898-891c6818-ec76-4d6b-b00d-a71d11b340e7.png)

Reply:
I just tried in Klayout 27.2 and I don't notice this issue. I have previously encountered an issue where the klayout database units where 0.001 and loading something in afterwards with a different set of units caused the layout to double in size. What version of klayout are you using?

Reply:
I'm using 26.2.

Reply:
Then I wonder if this is a bug in klayout since there were a lot of improvements to LEF/DEF handling from 26 to 27.

Reply:
Looks like you are correct Peter. After I upgraded to 27.2 the units converged.